taskboot/build.py

```diff
@@ -165,6 +166,19 @@ def build_hook(target, args):
     with open(hook_file_path) as hook_file:
         payload = json.load(hook_file)
 
+    # Inject environment variables if asked
+    payload_env = payload["task"]["payload"]["env"]
+    for env_variable in args.forward_env:
+        env_value = os.getenv(env_variable, default=None)
+
+        if env_value:
+            # TODO: Should we check if there is already an env variable with
+            # the same name defined?
```

Comment:
I would add a log for each variable being written, with a special case when it's **overwritten**, even with a warning level...